core/src/main/java/gyro/core/scope/NodeEvaluator.java

```diff
@@ -220,6 +220,10 @@ public static boolean test(Object value) {
     }
 
     public static Object getValue(Node node, Object object, String key) {
+        if (object != null && object.toString().equals("<output>")) {
```

Comment:
```suggestion
        if ("<output>".equals(object)) {
```

Further, make "<output>" a static field.

private static final String OUTPUT_VALUE = "<output>";